pkg/gds/gds.go

```diff
@@ -274,7 +274,11 @@ func (s *GDS) Lookup(ctx context.Context, in *api.LookupRequest) (out *api.Looku
 		}
 
 		if len(vasps) != 1 {
-			log.Warn().Str("common_name", in.CommonName).Int("nresults", len(vasps)).Msg("wrong number of VASPs returned from search")
+			// Don't warn when common name is not found, just when multiple results are returned
+			if len(vasps) > 1 {
+				log.Warn().Str("common_name", in.CommonName).Int("nresults", len(vasps)).Msg("multiple VASPs returned from common name search in lookup")
```

Comment:
This will reduce some of the warning alerts we get and also make the problem more apparent.

Reply:
🙌